.ocp-indent

```diff
@@ -1,13 +0,0 @@
-normal
```

Comment:
Why removing this config ? It looks close to the default but not quite.

ocp-indent is used by everyone not on VS code.

Reply:
I restore this file.